lib/smb2/packet/write_request.rb

```diff
@@ -23,5 +23,9 @@ class WriteRequest < Smb2::Packet
 
     rest :buffer
 
+    # @return [Symbol] a key in {Smb2::COMMANDS}
+    def self.command
```

Comment:
Change to `COMMAND = WRITE` if you move to [class-constant lookup](https://github.com/rapid7/smb2/pull/28/files#r28820364)